troubleshooting.md

```diff
@@ -16,6 +16,13 @@ When working with organizations in Copilot Workspace, you may encounter some com
   1. Log out of Copilot Workspace.
   2. Go through SAML auth in the browser by looking at, say, a repository of the organization
   3. Then log back into Copilot Workspace.
+ 
+- **Other known limitations to working within organizations:**
+  1. The enterprise (or org) must opt-in to Copilot feature previews.
```

Comment:
This us hood, but could you add a link to each of these to some other information on how to do these things?

Also do all enterprises use SAML auth?